### DISCUSSION TITLE
Skyline and Soni manía

### DISCUSSION THREAD

**Discussion by JwSh-1** [skyline-emu/skyline#256] [Q&A]
I come from discort / # Suport I see that they conducted a survey about Sonic mania, I wanted to know if what they are doing is that the game shows graphics on the screen?

 Or if they are improvements for performance in general?

 (I am not very clear since I am Spanish speaking and Google translator is not very precise ,,)

**Comment by Eretick** [comment]
Hi! I' just tried the latest build and see black screen only, with sound.

**Reply by JwSh-1** [reply]
If I understand it is something that they are working in the short term, so I understood they were going to try to start the Sonic game, they commented by 4 days ago, surely there are still a couple of weeks, ... ((I'm curious what opinions are on what we should do next:
1 one - Worse initial performance but Sonic Mania works earlier
2 two - Better initial performance but Sonic Mania working is delayed) asked him in discrim and won the 1.)🤔🤔😕❓
